.gitignore

```diff
@@ -18,3 +18,4 @@ CMakeLists.txt.user
 
 # VS Code system files
 .vscode/
+*.profraw
```

Comment:
Remove this. (I believe @hsorby has already discussed this before.)

Reply:
Done.

Reply:
Doesn't look done.